src/installing/README.md

```diff
@@ -61,7 +71,13 @@ connect:
 - Management console: `cu -l /dev/cuaU0 -s 115200`
 - Morello console: `cu -l /dev/cuaU2 -s 115200`
 
-However, commands and device names will vary by operating system.
+However, commands and device names will vary by operating system.  For
+Ubuntu we suggest:
+
+- Management console: `sudo screen /dev/ttyUSB0 115200`
+- Morello console: `sudo screen /dev/ttyUSB2 115200`
+
+To quit screen type `Ctrl-A` followed by `k`.
```

Comment:
Hm, seems like the difference is one window (kill) vs the entire session (quit), so both are probably fine, though it seems ^\ is used more in documentation that I can find for this kind of thing

Reply:
Another option would be `picocom -b 115200`, I use that for all my serial console needs and find the help message it prints on startup rather useful for beginners who otherwise wouldn't know how to exit the console.

Reply:
The problem with screen is that Debian and Ubuntu alter the bindings a little, including for quit.  To be honest, I included it because I had forgotten that it's not part of base Ubuntu, so I'll have a play with picocom and cu when I reinstall with the release version in the next day or two.  Ricardo did have some strange problem with cu, though, but perhaps that was just a glitch.

Reply:
picocom wins (cu setuids to uucp, which doesn't have access to the ports by default).  I'll update the commit soon.